data-pipeline/tests/unit/pre_processing/test_aar.py

```diff
@@ -6,114 +6,98 @@
 
 def test_aar_data_has_correct_output_columns(prepared_aar_data: pd.DataFrame):
     assert list(prepared_aar_data.columns) == [
+        "LA",
+        "Estab",
         "Academy UPIN",
-        "In year balance",
-        "PFI School",
-        "Trust UPIN",
```

Comment:
Wont we need the TrustUPIN for the trust table, which we use to link it to the BFR data?

Reply:
@colinbull Are we still using the trust UPIN in the BFR process?